spacy/pipeline/spancat.py

```diff
@@ -272,6 +272,24 @@ def predict(self, docs: Iterable[Doc]):
         scores = self.model.predict((docs, indices))  # type: ignore
         return indices, scores
 
+    def suggest_candidates(
+        self, docs: Iterable[Doc], spankey: str = "candidates"
+    ) -> List[Doc]:
+        """Use the spancat suggester to add a list of span candidates to a list of docs
+
+        docs (Iterable[Doc]): The documents to predict.
+        RETURNS: A list of Docs.
+
+        DOCS: WIP
+        """
+        suggester_output = self.suggester(docs, ops=self.model.ops)
+
+        for candidates, doc in zip(suggester_output, docs):
```

Comment:
`mypy` complains about `zip` not being able to work with `(Ragged, Iterable[Doc])`. I'm not sure whether we can find a quick elegant solution to this - if not you can also just add the directive `  # type: ignore` to the end of this line.

Reply:
I added the directive for now :smile: